Minor updates to default template for consistency and to add an image attribute

jdanish opened the pull request:
NOTE: changed default password to ncpower because danish felt weird to me ...

Commit:
Removed googlea code that was throwing an error on the creation of a new node.

Commit:
Added a someimage markdown node attribute

NOTE: changed the default network password to ncpower

Spaced some of the "Some XXXX" labels for the display for consistency

Probably we should be more consistent with camel case but ... it threw an error when I tried so I left it

jdanish commented:
@benloh I also removed some google auth code that was throwing an error on node creation in case that matters

benloh commented:
> I also removed some google auth code that was throwing an error on node creation in case that matters

Yeah, sorry about that.  Some earlier cleanup resulted in this.  You'll find a few other google analytics calls causing issues too.  The proper fix is 3b22ee88fe89f28b1d0f1216dc9454aaeb0538f7.

benloh commented:
@jdanish I can confirm that it validates and runs, but...

* Are you sure you want to sometimes use a space and sometimes not?  e.g. sometimes the displayLabel has a space ("Some Number") but the export label does not ("SomeNumber") but it's not consistent.
* I'm assuming you want the image field to be a Markdown field.  If that's the case, then the entry should be:
```
[nodeDefs.someimage]
type = "markdown".          // NOTE 'markdown', not 'string'
displayLabel = "Image"
exportLabel = "Image"
```

Commit:
Fixed some image should be markdown, not string
Changed someimage to image
For other two-part names, added space for consistency (e.g., Other Thing)
Added export label to all for consistency

jdanish commented:
Good catch - I had made the change but then lost it when I fixed formatting issues

So, now:

Image is Image across the board (no Some), and is markdown
Other two-part names (e.g., OtherText) always have a space now

In doing that, the inconsistent presence of export label was bugging me so it is now consistently present

benloh commented:
@jdanish Good catch re the missing `exportLabel` definitions.
